masu/processor/tasks.py

```diff
@@ -20,13 +20,14 @@
 # disabled module-wide due to current state of task signature.
 # we expect this situation to be temporary as we iterate on these details.
 
+import logging
+
 from celery import shared_task
-from celery.utils.log import get_task_logger
 
 from masu.processor._tasks.download import _get_report_files
 from masu.processor._tasks.process import _process_report_file
 
-LOG = get_task_logger(__name__)
+LOG = logging.getLogger(__name__)
```

Comment:
I'm pretty sure that using Celery's task_logger is the right thing here.  Can you confirm that this change is necessary?

Reply:
I’ll take a look at using get_task_loggrer and adjusting the level.   I don’t care how it’s done just want it the logs :)

Reply:
--loglevel=INFO worked like a charm, thanks guys for pointing that out!